README.md

```diff
@@ -13,7 +13,7 @@
 
 A collection of simplified and community-driven man pages.
 
-Install it with `npm install -g tldr` or [try the web client](http://tldr.ostera.io).
+Install it with `npm install -g tldr`, `brew install tldr` (on OSX, using [Homebrew](http://brew.sh)), or [try the web client](http://tldr.ostera.io).
```

Comment:
There would be a slight disparity if we mention a platform specific method with a platform independent method. 

For eg. I would be fine if `brew install` is mentioned along with `apt-get install`, or if just `npm install` is mentioned. But mentioning `brew install` along with `npm install` slightly gives the indication that for OSX platforms, one should use `brew install`, which is not true at all.